app/src/main/res/values/strings.xml

```diff
@@ -87,25 +87,25 @@
     <string name="action_show_solution">Show solution</string>
     <string name="action_show_mine_sudoku">Show mine</string>
     <string name="onboard_title">Welcome</string>
-    <string name="intro_what_is_sudoku">A classic sudoku looks as follows</string>
-    <string name="intro_rules">Use 1–9 to fill all cells—without repetition in any column, row or smaller (3x3) box.</string>
-    <string name="onboard_recommendation_prefs">Go through the settings to adjust the game</string>
+    <string name="intro_what_is_sudoku">A classic sudoku</string>
```

Comment:
The actual invocation of how it looks is ironically farther removed from where it is presented visually by both superfluous info and the text below it.
The game is translated into Russian, which often overshoots boundaries this way.

https://developer.android.com/docs/quality-guidelines/core-app-quality
`The app displays text and text blocks in an acceptable manner for each of the app’s supported languages. Composition is acceptable in all supported form factors.No cut-off letters or words are visible. No improper word wraps within buttons or icons are visible. There is sufficient spacing between text and surrounding elements.`

The end result (considering all changes) I think induces less stress.
It is a pivotal point in not losing userbase.

Reply:
I want to do as little of that as possible, in keeping with the stated intent.

Reply:
The total here is important, because these are the grimmest strings AFAICT.
If at all anyone complains that there isn't a connection between what it says and what is shown,
it can easily be changed back.

What is the entire thing trying to do, and is it doing anything else is where I can.
Ultimate length of text is only correlated.

There can be visual clue strings for impaired users. Like the ARIA HINT ones, but for Android.

https://support.google.com/accessibility/android/answer/7158690?hl=en
>If the element is associated with an action, describe the action, not the graphical representation.